sphinxlint/rst.py

```diff
@@ -145,10 +145,10 @@
 ROLE_TAG = rf":{SIMPLENAME}:"
 ROLE_HEAD = rf"({BEFORE_ROLE}:{SIMPLENAME}:)"  # A role, with a clean start
 
-ASCII_ALLOWED_BEFORE_INLINE_MARKUP = r"""[-:/'"<(\[{]"""
-UNICODE_ALLOWED_BEFORE_INLINE_MARKUP = r"[\p{Ps}\p{Pi}\p{Pf}\p{Pd}\p{Po}]"
-ASCII_ALLOWED_AFTER_INLINE_MARKUP = r"""[-.,:;!?/'")\]}>]"""
-UNICODE_ALLOWED_AFTER_INLINE_MARKUP = r"[\p{Pe}\p{Pi}\p{Pf}\p{Pd}\p{Po}]"
+ASCII_ALLOWED_BEFORE_INLINE_MARKUP = r"""-:/'"<(\[{"""
+UNICODE_ALLOWED_BEFORE_INLINE_MARKUP = r"\p{Ps}\p{Pi}\p{Pf}\p{Pd}\p{Po}"
+ASCII_ALLOWED_AFTER_INLINE_MARKUP = r"""-.,:;!?/'")\]}>"""
+UNICODE_ALLOWED_AFTER_INLINE_MARKUP = r"\p{Pe}\p{Pi}\p{Pf}\p{Pd}\p{Po}"
```

Comment:
FWIW all ASCII chars are technically already included in the Unicode categories, except for `<>` which have an `Sm` category.  Even though they are redundant, I think it's ok since it's more explicit and it's easier to see what (ASCII) characters are allowed.

Reply:
Also this is a copy-paste from the spec: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#inline-markup-recognition-rules

I like copy-pasting from the spec.